### PR TITLE
cohort builder

### PR DESCRIPTION
**Pull request by vsmalladi**
Updates to cohort builder

### CODE REVIEW

**Comment by mingjiecn** [comment]
1. The sizing is off in Firefox:
image

2. The image is not loaded in Chrome:
image

3. still need to log in twice.

4. since we changed the cart to cohort, should we change the naming rule for the name of cohort as well?
image

**Comment by mingjiecn** [comment]
Need to build biosample cohorts as well.

File: src/encoded/static/components/home.js [hunk 1]
```diff
@@ -641,16 +641,7 @@ export default class Home extends React.Component {
...
-                            <HomeBanner adminUser={adminUser} />
-                            <div className="organism-tabs">
-                                <TabClicking organisms={this.state.organisms} handleTabClick={this.handleTabClick} />
-                            </div>
-                            <div className="graphs">
-                                <div className="row">
-                                    <HomepageChartLoader organisms={this.state.organisms} assayCategory={this.state.assayCategory} query={currentQuery} />
-                                </div>
-                            </div>
+                        <img src="/static/img/banner-bg.jpg  " alt="UT Southwestern" id="su-logo" height="390"/>
```
Comment (SenseCode) [inline src/encoded/static/components/home.js:644]: banner-bg.jpg is not there. Or forget_ to copy in, or wrong route?

**Comment by SenseCode** [comment]
image
It's better to show done after finished process with the processing bar...
Another bug is if adding cohorts before log in, then all the cohorts are gone, cart is empty...

File: src/encoded/static/components/cart/util.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { Modal, ModalHeader, ModalBody, ModalFooter } from '../../libs/bootstrap
 
 /** List of object @type allowed in the cart. */
 const allowedCartTypes = [
-    'Experiment',
+    'Patient',
```
Comment (vsmalladi) [inline src/encoded/static/components/cart/util.js]: Need to add Biospecimen to link to openspecimen

Maybe all Searchable objects:
Surgery
PathReport
Biospecimen

**Comment by vsmalladi** [comment]
After making cohort then login chort is lost.

File: src/encoded/static/components/cart/util.js [hunk 3]
```diff
@@ -20,9 +20,9 @@ export const CART_MAXIMUM_ELEMENTS_LOGGEDOUT = 4000;
  */
 export const MaximumElementsLoggedoutModal = ({ closeClickHandler }) => (
     <Modal>
-        <ModalHeader title="Too many experiments selected" closeModal={closeClickHandler} />
+        <ModalHeader title="Too many patients selected" closeModal={closeClickHandler} />
         <ModalBody>
-            <p>You can add a maximum of {CART_MAXIMUM_ELEMENTS_LOGGEDOUT} patients to a cart if you have not logged in.</p>
+            <p>You can add a maximum of {CART_MAXIMUM_ELEMENTS_LOGGEDOUT} patients to a cohort if you have not logged in.</p>
```
Comment (SenseCode) [inline src/encoded/static/components/cart/util.js:25]: What the maximum numbers of elements we can add into cohort folder now?

File: src/encoded/static/components/home.js (hunk 1 above)
Comment (SenseCode) [reply src/encoded/static/components/home.js:644]: Home page pic works now. But not full screen..
image

File: src/encoded/static/components/cart/cart.js [hunk 4]
```diff
@@ -21,10 +21,10 @@ import CartMergeShared from './merge_shared';
...
 /** File facet fields to display in order of display */
 const displayedFacetFields = [
-    'output_type',
-    'file_type',
-    'assembly',
-    'lab.title',
+    'sample_type',
+    'tissue_derivatives',
+    'tissue_type',
+    'anatomic_site',
```
Comment (SenseCode) [inline src/encoded/static/components/cart/cart.js:27]: I didn't find these codes matters somewhere?

**Review by SenseCode — APPROVED** [review]
This cohort_builder works great. Solve all the issues we addressed. Great job..

**Review by mingjiecn — APPROVED** [review]
Fixed the bug I mentioned. Also fixed this issue #157

**Pull request merged**